packages/replay/src/replay.ts

```diff
@@ -942,6 +942,7 @@ export class ReplayContainer implements ReplayContainerInterface {
       trace_ids: traceIds,
       urls,
       replay_id: replayId,
+      event_id: replayId,
```

Comment:
I don't think this is what we want as we will then have multiple replay events with the same event id?

Reply:
I think we can get of this line, right? The uuid is [assigned in _prepareEvent](https://github.com/getsentry/sentry-javascript/blob/ed5c19098d83dd71348c34e730852f09ff63a15e/packages/core/src/baseclient.ts#L419-L426).

Reply:
done!